### ISSUE TITLE
[Feature] LFS support for Migrate and Mirror

### ISSUE THREAD

**Issue by 6543** [go-gitea/gitea#9939]
as title

**Comment by 6543** [comment]
```sh
# create a bare clone of the GitHub repository
git clone --bare git@github.com:repo_owner/repo_name.git
cd repo_name

# set up named remotes for Gitea and GitHub
git remote add gitea git@gitea.org:tpettersen/repo_name.git
git remote add github git@github.com:repo_owner/repo_name.git

# fetch all Git LFS content from GitHub
git lfs fetch --all github

# push all Git and Git LFS content to Gitea
git push --mirror gitea $ git lfs push --all gitea
```

**Comment by zeripath** [comment]
this would require git lfs to be installed with gitea or that we implement a git lfs client (which potentially wouldn't be too difficult)

BTW we can't do: 

```
# fetch all Git LFS content from GitHub
git lfs fetch --all github

# push all Git and Git LFS content to Gitea
git push --mirror gitea $ git lfs push --all gitea
```

we have to have the fetch deposit the assets directly into gitea's lfs store as soon as an LFS file is obtained.

**Comment by 6543** [comment]
isnt git-lfs implemented in go?

I think we need a own client to store things into our lfs directly

**Comment by KN4CK3R** [comment]
This issue can be closed with #14726 and #15157 I guess?
